src/common/ceph_context.h

```diff
@@ -65,7 +67,7 @@ class CephContext {
 	      int = 0)
     : CephContext{}
   {}
-  CephContext(CephContext&&) = default;
+  CephContext(CephContext&&); // = default;
```

Comment:
If you're changing the constructor, go ahead and just remove the = default part.

Reply:
fixed